docs/source/advanced.rst

```diff
@@ -25,8 +25,9 @@ Turn Off Private Key
 
 By default, all OnionShare services are protected with a private key, which Tor calls "client authentication".
 
-The Tor Browser will ask you to enter your private key when you load an OnionShare service.
-If you want allow the public to use your service, it's better to disable the private key altogether.
+The Tor Browser will ask you to enter your private key when you want to offer an OnionShare service.
+If you want to allow the public to use your service, it's better to turn off the private key so that people you don't know
+wont have to find your public key.
```

Comment:
Better would be 'so that people won't have to enter it'. Also not `wont` ought to be `won't`.